### PR TITLE
Map Tweaks: SCC Scout, Civ Yacht; Shuttle Window Spawners

### PR DESCRIPTION
**Pull request by DreamySkrell**
changes:
  - maptweak: "SCC Scout Ship mapping and name tweaks."
  - maptweak: "Civilian Yacht mapping tweaks."
  - rscadd: "Shuttle window spawners."

### CODE REVIEW

**Comment by DreamySkrell** [comment]
!wip

File: code/modules/effects/map_effects/window_spawner.dm [hunk 1]
```diff
@@ -1,19 +1,28 @@
-//
-// Window Spawner
-//
+
+// ----------------------------------- window spawner base
```
Comment (SleepyGemmy) [inline code/modules/effects/map_effects/window_spawner.dm:2]:
```suggestion
// ----------------------------------- Window Spawner Base
```

File: code/modules/effects/map_effects/window_spawner.dm [hunk 2]
```diff
@@ -78,7 +89,8 @@
 /obj/effect/map_effect/window_spawner/proc/handle_grille_spawn(var/obj/structure/grille/G)
 	return
 
-/********** Quarter Windows **********/
+// ----------------------------------- old quarter windows subtypes
```
Comment (SleepyGemmy) [inline code/modules/effects/map_effects/window_spawner.dm:92]:
```suggestion
// ----------------------------------- Old Quarter Windows Subtypes
```

File: code/modules/effects/map_effects/window_spawner.dm [hunk 3]
```diff
@@ -128,7 +140,8 @@
 	if(id)
 		W.id = id
 
-/********** Full Windows **********/
+// ----------------------------------- windows subtypes
```
Comment (SleepyGemmy) [inline code/modules/effects/map_effects/window_spawner.dm:143]:
```suggestion
// ----------------------------------- Windows Subtypes
```

Comment (DreamySkrell) [reply code/modules/effects/map_effects/window_spawner.dm:143]: I think it's fine as is

File: code/modules/effects/map_effects/window_spawner.dm (hunk 2 above)
Comment (DreamySkrell) [reply code/modules/effects/map_effects/window_spawner.dm:92]: I think it's fine as is

File: code/modules/effects/map_effects/window_spawner.dm (hunk 1 above)
Comment (DreamySkrell) [reply code/modules/effects/map_effects/window_spawner.dm:2]: I think it's fine as is

**Comment by DreamySkrell** [comment]
!review

**Review by NonQueueingMatt — APPROVED** [review]

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**